src/main/java/features/Storage.java

```diff
@@ -0,0 +1,92 @@
+package features;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import tasks.Deadline;
+import tasks.Event;
+import tasks.ToDo;
+
+/**
+ * Saves and loads state of taskList to/from data/duke.txt.
+ */
+public class Storage {
+
+    /**
+     * Loads taskList from data/duke.txt, if any.
+     * @return the loaded taskList.
+     */
+    public TaskList loadTaskList() {
+
+        TaskList toLoad = new TaskList();
+        File dukePath = new File("data");
+        File dukeSave = new File("data/duke.txt");
+
+        try {
+            if (!dukePath.exists()) {
+                if (dukePath.mkdirs()) {
+                    System.out.println("No save files detected. Creating new save file...");
+                }
+                if (!dukeSave.exists()) {
+                    if (dukeSave.createNewFile()) {
+                        System.out.println("New save file created!");
+                    }
+                }
+            }
+        } catch (IOException err) {
+            System.out.println("Sorry, could not save that.");
+        }
+
+        try {
+            Scanner fileScan = new Scanner(dukeSave);
+            while (fileScan.hasNextLine()) {
+                String toEval = fileScan.nextLine();
```

Comment:
Similar case, `toEval` can be rephrased as a noun.

Reply:
Same as above, i've made the change now.